src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -170,6 +172,7 @@ const usePushNotificationHook = () => {
     if (localDb) {
       messaging().setBackgroundMessageHandler(async (remoteMessage) => {
         if (remoteMessage.data?.type === 'message.new') {
+          setIsLoadingFetchAllMessage(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line sets the loading state to true before fetching messages. It's good practice to handle loading states for async operations to improve user experience. However, ensure that error handling is also implemented to set loading state back to false in case of any errors during fetch operation.

```diff
+ setIsLoadingFetchAllMessage(true);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->